ConnectivityTestScriptsForWindows/Start-ConnectivityTests.ps1

```diff
@@ -154,12 +154,12 @@ try
 
                     $error.clear()
                     $privateip = $null
-                    $privateip = Resolve-DnsName $serviceUrl
+                    $privateip = Resolve-DnsName $serviceUrl -ErrorAction SilentlyContinue
 
                     if( $privateip -eq $null)
                     {
-                        TraceMessage "`nFailed to resolve $serviceUrl"  "Red"
-                        $dns.PrivateIp = $error
+                        TraceMessage "`nFailed to resolve1 $serviceUrl"  "Red"
```

Comment:
why was this change required, resolve -> resolve1

Reply:
fixed